cmd/ferretdb/main.go

```diff
@@ -521,7 +525,9 @@ func run() {
 
 	metricsRegisterer.MustRegister(l)
 
+	listenerStarted.Store(true)
```

Comment:
I see the PR is merged, but I still have a question.

At that moment, the listener hasn't started yet, right?
With the current approach, what is the difference between having this `listenerStarted` var and always returning `ok` in the probe? How to reproduce a moment when the probe doesn't return `ok`?

Reply:
> At that moment, the listener hasn't started yet, right?

No, that's not right in the current code